ExtractLightsVariance

```diff
@@ -51,9 +51,57 @@ struct light {
 };
 
 
-double luminance(double r, double g, double b)
+static double luminance(double r, double g, double b)
 {
     return r*0.2125f + g*0.7154f + b*0.0721f;
 }
 
+// solid Angle
+static double AreaElement( const double x, const double y )
+{
+    return atan2(x * y, sqrt(x * x + y * y + 1.0));
+}
+
+// a pixel to a solidAngle
+static double texelAreaSolidAngle(const double aU, const double aV, const double aW, const double aH, const uint width, const uint height)
```

Comment:
note the same, one comes is for cubemap and use double, the other (here) is for equirect map and in double. 
Can move In Marth
Need Rename to avoid confusion too.
"texelAreaSolidAngleEquRect"